src/microbe_stage/components/Engulfer.cs

```diff
@@ -189,6 +190,13 @@ public static EngulfCheckResult CanEngulfObject(this ref Engulfer engulfer, uint
             if (!compounds.Compounds.Any(p => usefulCompoundSource.IsUseful(p.Key)))
                 continue;
 
+            // Skip entities that are too small to catch easily
+            if (skipLikelyTooFastTargets &&
+                engulfer.EngulfingSize / Constants.TUTORIAL_ENGULFABLE_SIZE_RATIO > engulfable.AdjustedEngulfSize)
+                {
```

Comment:
Well I missed up here with the Github code editor.

I'll pull the changes locally to fix this and do some testing if the gameplay change is now good.